app/controllers/assignments_controller.rb

```diff
@@ -73,29 +71,31 @@ def create
                        .permit :start_date, :end_date,
                                :user_id, :roster_id
     assignment = Assignment.new ass_params
```

Comment:
This needs to become `@assignment` for the form to rerender correctly on error, and this action needs to be added to the `set_roster_users` before action list